common/reducers/Filter.js

```diff
@@ -11,7 +11,8 @@ module.exports = function Filter(prevState = INITIAL_STATE, action) {
   switch (action.type) {
```

Comment:
I belatedly remembered that you can't put a declaration inside a case block.  That said, you could declare it using let above the switch, and Object.assign inside the case block.  Totally a micro-optimization though, so it's entirely up to you whether you do it.  If you do so, I'd suggest adding a comment explaining why so that some future maintainer doesn't undo it.